### PR TITLE
fix #82065 fullhdfilmizlesenebox.com

### LINKED ISSUE #82065: www.fullhdfilmizlesenebox.com
### Issue URL (Ads)
 [https://www.fullhdfilmizlesenebox.com/onemsiz-biri-izle/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.fullhdfilmizlesenebox.com%2Fonemsiz-biri-izle%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#82065

### CODE REVIEW

File: TurkishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -346,6 +346,8 @@ r10.net#%#document.cookie = 'r10LeftHide=1';
 !------- CSS fixes --------------------!
 !--------------------------------------!
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/82065
+fullhdfilmizlesenebox.com#$#body { background: #000 !important; }
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/general_extensions.txt:350]: better:
`||fullhdfilmizlesenebox.com/wp-content/uploads/*/Pageskin.jpg`

File: SpywareFilter/sections/tracking_servers_firstparty.txt [hunk 2]
```diff
@@ -55,6 +55,7 @@
 ||stat.5-tv.ru^
 ||log.zingnews.vn^
 ||logcol.kyobobook.co.kr^
+||ping.staticmoly.me^
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/tracking_servers_firstparty.txt:58]: please move to tracking_servers.txt

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply TurkishFilter/sections/general_extensions.txt:350]: You are a magician.

Comment (wout-codes) [reply TurkishFilter/sections/general_extensions.txt:350]: I have a history of making things complicated.

**Pull request merged**